src/tests/accommodation.test.js

```diff
@@ -2,7 +2,7 @@ import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 import { describe, it } from 'mocha';
 import app from '../index';
-import {
+import { // c8e9428a-6d60-4083-8ce3-334c62afe72c
```

Comment:
```suggestion
import {
```
@RusimbiP Why are you commenting this ```// c8e9428a-6d60-4083-8ce3-334c62afe72c``` out? Would you clean the code if you don't need that line of code.

Reply:
Alright, thanks